### ISSUE TITLE
[aws for fluentbit] support pod labels in the chart

### ISSUE THREAD

**Issue by nwsparks** [aws/eks-charts#1008]
**Is your feature request related to a problem?**

Sort of. Without being able to add pod labels to the pods this chart creates we cannot use templated log group names with it and it results in us having to exclude these pods from logging.

**Is your feature request related to a specific Helm chart, if yes mention name of the chart?**

aws for fluentbit

**Describe the solution you'd like**

add a podLabels value to the helm chart

**Comment by andrewhharmon** [comment]
agreed, this seems pretty standard to allow for adding additional labels to pods for usage/cost reporting as well
